Tools/TestWebKitAPI/Tests/WebKitGtk/TestWebViewEditor.cpp

```diff
@@ -271,6 +271,24 @@ class EditorTest: public WebViewTest {
         return typingAttributes();
     }
 
+    static gboolean editorStateChangedCallback(EditorTest* test)
```

Comment:
```suggestion
    static void editorStateChangedCallback(EditorTest* test)
```

Reply:
(You defined the signal as returning void, so your signal handler shouldn't have any return value.)

Reply:
Done.